packages/styles/components/_c.button.scss

```diff
@@ -50,9 +53,11 @@
     }
 
     @each $size, $props in $icons {
```

Comment:
I thought that each button had a dedicated icon size ?

Reply:
fixed